requirements/edx/github.in

```diff
@@ -93,6 +93,7 @@ git+https://github.com/edx/xblock-lti-consumer.git@v1.1.8#egg=lti_consumer-xbloc
 
 # Third Party XBlocks
 
+git+https://github.com/joestump/python-oauth2.git@master#egg=oauth2
```

Comment:
Instead of `master` use the hash of master.

Reply:
@feanil yes `make upgrade` did fail at first, but if we update the `Django` version in `django-oauth-plus` repository (as done in this PR https://github.com/edx/django-oauth-plus/pull/5) and provide the hash from this PR then it works fine, should we go ahead with this?

Reply:
Yes, let's do that.